frontend/src/libs/components/icon/icon.tsx

```diff
@@ -0,0 +1,22 @@
+import { type IconNames } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { iconNameToPlainSvgMap } from '#libs/maps/icon-name-to-plain-svg.map.js';
```

Comment:
It's better to create `libs/aps/maps.ts` to reexport all maps from there

Reply:
We've been covering where to move maps in [cp-61](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/75#discussion_r1310513955)